### PR TITLE
[Website]: mobile nav fix #27 fix

### LINKED ISSUE #27: [Website]: Mobile menu not correct
## Issue type

* Website

## What happened

When making the browser narrow to replicate a mobile experience:

1. The menu icon appears.
2. The menu can be opened and closed.
3. The menu is incorrect - still using the one from the USWDS.

## What I expected

The menu would replicate the desktop experience and function similar to the USWDS.

## Steps to reproduce the issue [optional]

1. Go to site on mobile device (or narrow browser window).
2. Tap/Click "menu".
3. Review navigation links.

## Additional information [optional]

screen shot 2016-08-01 at 8 35 41 am

### PR DESCRIPTION
**Pull request by diego-ruiz-rei**
## Description

Mobile nav fix for issue #27 

## Additional information

Before

screen shot 2016-08-16 at 11 21 18 am

After

screen shot 2016-08-16 at 11 21 01 am

### CODE REVIEW

File: assets/img/close.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (joshbruce) [inline assets/img/close.svg]: This is awesome! Thank you.

If memory serves - this image is an X from the USWDS. What do you think of using the "dismiss" button instead?

https://gsa.github.io/sam-web-design-standards/elements/buttons/

**Comment by diego-ruiz-rei** [comment]
yep I agree - seeing as it is an actual defined button in the standards.

File: _includes/site-navigation/sidenav-mobile.html [hunk 2]
```diff
@@ -1,126 +1,133 @@
 {% assign current = page.url | downcase | split: '/' %}
 
 <aside class="sidenav-mobile">
-  <a class="sliding-panel-close" href="#">
-    <img src="/assets/img/close.svg" alt="close">
-  </a>
+  <button class="usa-button-dismiss sliding-panel-close"> <span class="usa-sr-only">Dismiss</span></button>
   <nav>
```
Comment (joshbruce) [inline _includes/site-navigation/sidenav-mobile.html]: @diego-ruiz-rei - What's the rationale behind switching to the `button` element? Legitimately curious.

Comment (diego-ruiz-rei) [inline _includes/site-navigation/sidenav-mobile.html]: mostly copy+pasted from the code sample in the samwds button documentation, it has the .usa-button-dismiss class added to get the svg/png images as a background image

**Comment by diego-ruiz-rei** [comment]
from our discussion, updated to use anchor tag over button

File: _includes/site-navigation/sidenav-mobile.html [hunk 3]
```diff
@@ -1,126 +1,133 @@
 {% assign current = page.url | downcase | split: '/' %}
 
 <aside class="sidenav-mobile">
-  <a class="sliding-panel-close" href="#">
-    <img src="/assets/img/close.svg" alt="close">
-  </a>
+  <a class="sliding-panel-close usa-button-dismiss" href="#"></a>
```
Comment (joshbruce) [inline _includes/site-navigation/sidenav-mobile.html]: @diego-ruiz-rei - Thanks! Sorry, I should have been more specific. To meet 1194.22(L) we need to offer a text-based alternative as well; that's where the `span` comes into play: `<span class="usa-sr-only">Close</span>`

The `usa-sr-only` class moves the text off-screen, but maintains it as a point for screen readers - hence the `sr-only`. Hope there clarifies.

Thanks again!

Comment (diego-ruiz-rei) [inline _includes/site-navigation/sidenav-mobile.html]: gotcha, added in

**Comment by joshbruce** [comment]
LGTM! Cheers.

**Pull request merged**